### PR TITLE
Fixes #25 - Accept Shortened YouTube Links

### LINKED ISSUE #25: Accept Shortened YouTube Links

### PR DESCRIPTION
**Pull request by Createor**
Adds the ability to match shortened video links

### CODE REVIEW

File: api/services/YouTubeService.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
 };
 
 function parseYouTubeLink(link) {
-  var match = link.match(/v=[^?&]+/g);
+  var match = link.match(/v=[^?&]+|(?:youtu.be\/).+/g);
```
Comment (SirZach) [inline api/services/YouTubeService.js]: @Createor we need to make sure that youtube's API also accepts shortened urls

Comment (TheConnMan) [inline api/services/YouTubeService.js]: We only need the key and thankfully the key is the same across both links.

**Comment by Createor** [comment]
Anything else I left out?

**Comment by TheConnMan** [comment]
G2G

**Pull request merged**